docs/docs/install/docker/index.md

```diff
@@ -0,0 +1,146 @@
+# Install Mathesar via Docker
+
+
+## Prerequisites
+!!! warning ""
+    This installation procedure is intended for users who want to run a bare-bones version of the Mathesar Webserver. It is assumed you already have services like a reverse proxy needed for running a production server. Please look at [Docker Compose Installation Documentation](../docker-compose/index.md) for running services needed for a proper production server 
+
+- You need [Docker](https://docs.docker.com/get-docker/)
+  - We have tested with Docker v23. Older versions may not work.
+- Permission to run Docker containers
+- A Postgres database. Ensure the external database can accept network connections from your Mathesar server.
+- Have the following information handy before installation:
+    - Database hostname
+    - Database port
+    - Database name
+    - Database username _(should exist and be a `SUPERUSER` [more info](https://www.postgresql.org/docs/13/sql-createrole.html))_
+    - Database password
+
+## Installation Steps
+
+1. Run the Mathesar Docker Image
+    ```bash
+    docker run \
+      --detach
+      -e DJANGO_DATABASE_URL='<replace with a postgres connection string>' \
+      -e DJANGO_DATABASE_KEY='default' \
+      -e MATHESAR_DATABASES='(<unique_db_key>|<replace with a postgres connection array>)' \
+      -e SECRET_KEY='<replace with a 50 character string>' \
+      -e ALLOWED_HOSTS='.localhost, 127.0.0.1, [::1]' \
+      -e DJANGO_SETTINGS_MODULE='config.settings.production' \
```

Comment:
Are `DJANGO_DATABASE_KEY` and `DJANGO_SETTINGS_MODULE ` essential here?

I think the more minimal the command is, the better it is. Anyone who wants to customize, can refer the configuration guidelines.